src/ActiveLogin.Authentication.BankId.AspNetCore/LoggerExtensions.cs

```diff
@@ -31,6 +31,11 @@ public static void BankIdAuthSuccess(this ILogger logger, SwedishPersonalIdentit
             logger.LogTrace(BankIdLoggingEvents.BankIdAuthSuccess, "BankID auth succedded for PersonalIdentityNumber '{PersonalIdentityNumber}' with the OrderRef '{OrderRef}'", personalIdentityNumber?.To12DigitString() ?? MissingPersonalIdentityNumber, orderRef);
         }
 
+        public static void BankIdAuthCancelled(this ILogger logger, string orderRef)
+        {
+            logger.LogInformation(BankIdLoggingEvents.BankIdAuthSuccess, "BankID auth was cancelled with the OrderRef '{OrderRef}'", orderRef);
```

Comment:
This is probably a copy paste mistake.

Would like to have the event like this in `BankIdLoggingEvents.cs`:
`public static readonly EventId BankIdAuthHardCancel = new EventId(2_1_3, nameof(BankIdAuthHardCancel));`

Reply:
Very nice, didn't see those.